cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,129 @@
+- [Egress Perimeter Controls](egress-perimeter-controls.html) must be enabled, Egress Perimeter Controls ensure that cluster egress operations, such as customer-managed cluster backups or changefeeds, are restricted to a list of specified external destinations.
+- [Cluster log export](export-logs.html)'s redaction feature must be enabled to prevent the exposure of sensitive data in logs exported to the customer's instance of AWS Cloudwatch or GCP Cloud Logging.
+- [Cloud Organization audit logs](cloud-org-audit-logs.html) automatically capture information when many types of events occur in your {{ site.data.products.db }} organization, such as when a cluster is created or when a member is added to or removed from an organization.
+- Cockroach Labs recommends enabling the following Single Sign-On (SSO) features, which helps you minimize the risk of password exposure in {{ site.data.products.db }}:
+  - [Cloud Organization SSO](configure-cloud-org-sso.html) allows members of your CockroachDB Cloud organization to authenticate to CockroachDB Cloud using an identity from an identity provider (IdP). This integration can be done using SAML or OIDC.
+  - [Cluster SSO](cloud-sso-sql.html) allows users to connect to the SQL interface of a CockroachDB cluster using a JWT (JSON Web Token). A variety of JWT issuers can be configured, including CockroachDB Cloud (for connectivity using cloud CLI), GCP service accounts, Azure managed identities, and others.
+
+Cockroach Labs cannot provide specific advice about ensuring end-to-end compliance of your overall system with PCI DSS or how to implement a specific requirement across all operating environments. The following points help to illustrate some steps that organizations might take to ensure compliance, and is not exhaustive.
+
+- Safeguard cardholder data by a combination of encryption, hashing, masking, and truncation. For an example implementation, refer to [Integrate {{ site.data.products.dedicated }} with Satori](/docs/{{site.versions["stable"]}}/satori-integration.html).
+- Protect unencrypted cryptographic materials  both at rest and in transit. Restrict access to unencrypted key materials on a "need to know" basis.
+- Restrict access to tables and views that contain cardholder data on a "need to know basis" basis. Refer to [Authorization](/docs/{{site.versions["stable"]}}/authorization.html).
+- Take steps to protect networks that transmit cardholder data from malicious access over the public internet. Refer to [Network Authorization](network-authorization.html).
+- Regularly test all systems, applications, and dependencies that are involved in storing or processing cardholder data for known vulnerabilities.
+
+## See also
+
+- [Integrate {{ site.data.products.dedicated }} with Satori](/docs/{{site.versions["stable"]}}/satori-integration.html)
```

Comment:
Maybe add `[Authorization](/docs/{{site.versions["stable"]}}/authorization.html)` to this list?